cli/base.ts

```diff
@@ -0,0 +1,20 @@
+import { flags, Command } from '@oclif/command'
+
+enum EnvNodes  {
```

Comment:
potentially temporary solution if we can "plugin" to `celocli` and share config.